tests/Vertex/VertexTest.php

```diff
@@ -175,7 +175,7 @@ public function coordinatesAndExpectedInitialCardinalProvider()
             array(
                 array('28.8234055', '1.3072664'),
                 array('43.296482', '5.36978'),
-                'N'
+                'NNE'
```

Comment:
Is this change related?

Reply:
Yes, the direction changed a little bit for this test.